examples/vue/src/pages/ui/components/authenticator/custom-slots/index.vue

```diff
@@ -0,0 +1,92 @@
+<script setup lang="ts">
+import Amplify from 'aws-amplify';
+import {
+  Authenticator,
+  // Vue Composable to get access to validation errors
+  useAuthenticator,
+} from '@aws-amplify/ui-vue';
+import '@aws-amplify/ui-vue/styles.css';
+import { toRefs } from 'vue';
+
+import awsExports from '@environments/auth-with-federated/src/aws-exports';
+Amplify.configure(awsExports);
+
+const { toResetPassword, toSignIn } = toRefs(useAuthenticator());
+</script>
+
+<template>
+  <authenticator>
+    <template v-slot:header>
+      <div style="padding: var(--amplify-space-large); text-align: center">
+        <img
+          class="amplify-image"
+          alt="Amplify logo"
+          src="https://docs.amplify.aws/assets/logo-dark.svg"
+        />
+      </div>
+    </template>
+
+    <template v-slot:sign-in-header>
+      <h3
+        class="amplify-heading"
+        style="padding: var(--amplify-space-xl) 0 0 var(--amplify-space-xl)"
+      >
+        Sign in to your account
+      </h3>
+    </template>
+
+    <template v-slot:sign-in-footer>
+      <div style="text-align: center">
+        <button
+          @click="toResetPassword"
```

Comment:
@ErikCH I didn't see a need to `@click.prevent` since it's a `type="button"`

Reply:
That's fine!